Formula/libgxps.rb

```diff
@@ -27,8 +27,8 @@ class Libgxps < Formula
   depends_on "meson" => :build
   depends_on "ninja" => :build
   depends_on "pkg-config" => :build
+  depends_on "cairo"
   depends_on "glib"
-  depends_on "gtk+3"
```

Comment:
`gtk+3` was used for building tests (https://gitlab.gnome.org/GNOME/libgxps/-/blob/0.3.2/meson.build#L243-244)

`cairo` is runtime dependency indirectly linked via `gtk+3`